### DISCUSSION TITLE
About remove searchResolutionsOrderArray from FileUtils

### DISCUSSION THREAD

**Discussion by halx99** [axmolengine/axmol#941] [Polls]
Affect APIs

- `FileUtils::setSearchResolutionsOrder`
- `FileUtils::addSearchResolutionsOrder`
- `FileUtils::getSearchResolutionsOrder`

We’ll keep this discussion open for two weeks to gather feedback.

**Comment by rh101** [comment]
I can't find any references to it in any of my own project code, so I can't really remark on whether it is of any use.  In what situations would it be useful?

Would it be of any benefit for use in downloadable resource updates for resources that are supplied with the app release (like in an APK)? For instance, where you would set the resolution order to be from the newer downloads to what is already contained in the resource folder, so that the newer file is used instead of the one contained in the original resource folder.

EDIT: I think I use `FileUtils::addSearchPath()` for the example I mentioned above, although I'm not sure if that is even the best way to deal with it.

**Reply by halx99** [reply]
I think addSearchPath is enough, and I never use it in my career.
